en/docs/learn/jwe-payload-processing-policy.md

```
@@ -86,6 +86,6 @@ Upload the `jwePayloadProcessingRequestPolicy.j2` policy file which resides insi
 |----------------|--------------|-------------| -------- | ------ | --------------- |
 | applicationServiceBasicAuthCredentials | Application Service Basic Auth Credentials | Base64 encoded(admin-username:admin-password) basic auth credentials required to access the application service | true     | String | aXNfYWRtaW5Ad3NvMi5jb206d3NvMjEyMw== |
 | identityServerBaseUrl | Identity Server Base URL | Base URL of the identity server | true | String | https://localhost:9446 |
-| jweEncryptionAlg | JWE Encrytion Algorithm | Algorithm that is supported to use in encrypting the response paylaod.  | true | String | RSA-OAEP-256, RSA-OAEP, RSA-OAEP-384, RSA-OAEP-512, RSA1_5 |                    |
-| jweEncryptionMethod | JWE Encrytion Method | Encryption Method that is supported to use in encrypting the response paylaod. | true  | String | A128GCM, A256GCM, A192GCM |
-| jweEncryptionCertAlias | JWE Encrytion Certificate Alias | Alias of the private key of the server that is supported to use in encrypting the request paylaod. | true  | String | wso2carbon |
+| jweEncryptionAlg | JWE Encryption Algorithm | Algorithm that is supported to use in encrypting the response payload.  | true | String | RSA-OAEP-256, RSA-OAEP, RSA-OAEP-384, RSA-OAEP-512, RSA1_5 |                    |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 6; Actual: 7; Too many cells, extra data will be missing

(MD056, table-column-count)